src/main/cliapp/src/components/EvidenceEditor.js

```diff
@@ -10,20 +10,16 @@ import {AutoComplete} from "primereact/autocomplete";
             autocompleteFields.forEach( field => {
                 evidenceFilter[field] = event.query;
             });
+            let obsoleteFilter = {"obsolete": false};
 
-            searchService.search("ecoterm", 15, 0, null, {"evidenceFilter":evidenceFilter})
+            searchService.search("ecoterm", 15, 0, null, {"evidenceFilter":evidenceFilter, "obsoleteFilter:":obsoleteFilter})
                 .then((data) => {
-                    console.log(data);
-                    if (data.results) {
-                        setFilteredEvidenceCodes(data.results.filter((ecoterm) => Boolean(!ecoterm.obsolete)));
-                    }
-                    else {
-                        setFilteredEvidenceCodes([]);
-                    }
+                    console.log(data)
```

Comment:
Can we comment this console.log out? we try to get rid of those unless we are diagnosing an error or something.